spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -715,6 +720,7 @@ def create_client_for_kernel(
         sshkey=None,
         password=None,
         server_id=None,
+        give_focus=False,
```

Comment:
Please add `give_focus` to the docstring of this method. You can simply copy/paste the text for it in `create_new_client`.